asm/cluster/nodepool.yaml

```diff
@@ -24,25 +24,8 @@ spec:
     minNodeCount: 2
     maxNodeCount: 4 # {"type":"integer","x-kustomize":{"setter":{"name":"max-nodes","value":"4"}}}
   nodeConfig:
-    diskSizeGb: 100
-    diskType: pd-standard
     machineType: e2-standard-4
-    preemptible: false
-    oauthScopes:
-    - "https://www.googleapis.com/auth/cloud-platform"
-    - "https://www.googleapis.com/auth/compute"
-    - "https://www.googleapis.com/auth/devstorage.read_only"
-    - "https://www.googleapis.com/auth/logging.write"
-    - "https://www.googleapis.com/auth/monitoring"
-    - "https://www.googleapis.com/auth/service.management.readonly"
-    - "https://www.googleapis.com/auth/servicecontrol"
-    - "https://www.googleapis.com/auth/trace.append"
-    metadata:
-      disable-legacy-endpoints: "true"
     workloadMetadataConfig:
       nodeMetadata: GKE_METADATA_SERVER
-  management:
-    autoRepair: true
-    autoUpgrade: true
```

Comment:
I think default is `false` for these two. Which may still be perfectly fine, but this is a small change. From what I can see, the manual docs never had this on, so I'm not worried.

Reply:
`autoRepair` defaults to true for COS containers. But I believe these settings came from the trial.